public/app/routes/routes.ts

```diff
@@ -314,16 +314,19 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
     .when('/plugins', {
       template: '<react-container />',
       resolve: {
+        roles: () => ['Admin'],
         component: () => PluginListPage,
       },
     })
     .when('/plugins/:pluginId/', {
       template: '<react-container />',
       reloadOnSearch: false, // tabs from query parameters
       resolve: {
+        roles: () => ['Admin'],
         component: () => PluginPage,
       },
     })
+    // TODO: should also be accessible only by admins but not sure how to do ACL with angular routes.
```

Comment:
@torkelo do you have any pointers for this one?

Reply:
If you check explore route, it should have that